src/mantine-core/src/Tabs/Tab/Tab.tsx

```diff
@@ -54,6 +54,8 @@ export const Tab = forwardRef<HTMLButtonElement, TabProps>((props, ref) => {
   );
 
   const isActive = value === ctx.value;
+  const panelId = ctx.getPanelId(value);
+  const ariaControlsValue = document.getElementById(panelId) ? panelId : undefined;
```

Comment:
This will throw an error during server side rendering.

Reply:
You can use ModalBase component as reference, it has similar logic

Reply:
I will take a look, thanks

Reply:
@rtivital I added a mounted panel list to its provider, so document.getElementById is not necessary now. Based on your recommendation of ModalBase

Reply:
Fixed eslint errors after test.